coded_tools/intranet_agents_with_tools/absence_manager.py

```diff
@@ -232,12 +232,9 @@ def post_cancel_absence_details(
 
 # Example usage:
 if __name__ == "__main__":
-    CLIENT_ID = "XXX"  # Replace with your client_id
-    CLIENT_SECRET = "XXX"  # Replace with your client_id
```

Comment:
Removing that trap: we do not want someone to hardcode secrets here and inadvertently check them in.